AWSSDKSwiftCLI/Sources/AWSCLIUtils/Version.swift

```diff
@@ -7,15 +7,46 @@
 
 import Foundation
 import ArgumentParser
-import PackageDescription
 
-// MARK: - Decodable
+public struct Version: Equatable {
+    public let major: Int
+    public let minor: Int
+    public let patch: Int
+
+    private var versionString: String { "\(major).\(minor).\(patch)" }
+
+    init(_ major: Int, _ minor: Int, _ patch: Int) {
+        self.major = major
+        self.minor = minor
+        self.patch = patch
+    }
+
+    public init(_ value: String) throws {
+        let components = value.split(separator: ".")
+        guard components.count == 3 else {
+            throw Error("Version does not have three components")
+        }
+        guard let major = Int(components[0]), let minor = Int(components[1]), let patch = Int(components[2]) else {
+            throw Error("Version components are not all Int")
+        }
+        self.init(major, minor, patch)
+    }
+}
+
+extension Version: CustomStringConvertible {
```

Comment:
Like the previous `Version`, this struct provides the version string when printed.